packages/block-library/src/navigation/index.php

```diff
@@ -89,14 +89,13 @@ function block_core_navigation_build_css_font_sizes( $attributes ) {
  * Renders a Navigation Block derived from data from the theme_location assigned
  * via the block attribute 'location'.
  *
- * If the theme doesn't explicity support 'block-nav-menus' or no location was provided
- * as a block attribute then an empty string is returned.
+ * If no location was provided as a block attribute then false is returned.
  *
  * @param  array $location The location of the classic menu to display.
  * @param  array $attributes Navigation block attributes.
  * @return string|false HTML markup of a generated Navigation Block or false if no location is specified.
  */
-function render_classic_location_menu( $location, $attributes ) {
+function gutenberg_render_menu_from_location( $location, $attributes ) {
```

Comment:
@scruffian Just noticed that these two lines look to be unused in the function as well:
```php
$block_attributes = $attributes;
unset( $block_attributes['__unstableLocation'] );
```

And it looks like the second `$attributes` param/argument to the function can be dropped too.

Reply:
You're right, thanks. Done!